spyder/config/lsp.py

```diff
@@ -66,7 +66,7 @@
                 },
                 'jedi_completion': {
                     'enabled': True,
-                    'include_params': False
+                    'include_params': True
```

Comment:
This needs to be configurable and it also need to be False, to not confuse newbies by giving them a *strange* result as a completion.

Please add the option in

    Preferences > Completion and linting > Introspection > Basic features

with the name

    Enable snippet completions

Reply:
I think this option should be set to True by default, at least that's what other editors do, e.g., VSCode

Reply:
This is still open to discussion, but for now please deactivate this. We can see later if we want to activate it or not.

Reply:
@andfoy, please also address this.